packages/next/src/server/lib/incremental-cache/file-system-cache.ts

```diff
@@ -368,14 +380,18 @@ export default class FileSystemCache implements CacheHandler {
       value: data,
       lastModified: Date.now(),
     })
+
     if (this.debug) {
       console.log('set', key)
     }
 
-    if (!this.flushToDisk) return
+    if (!this.flushToDisk || !data) return
```

Comment:
is it possible for data to be falsy but we still want to write it to disk? ie `null` vs `undefined` vs `''` vs `false`

Reply:
It should not no. All the previous cases in the code here did not write anything when the data was not defined.

Reply:
Ahhh right I forgot the actual data is in `data.data` and not `data` 😆